integreat_cms/static/src/css/style.scss

```
@@ -53,6 +53,7 @@ header {
 nav#primary-navigation {
   #menu {
     max-height: calc(100vh - 50px);
```

```suggestion
    max-height: calc(100vh - 56px);
```
The logo has a height of 56px, so we should calculate the correct value here.